package.json

```diff
@@ -43,13 +43,14 @@
   },
   "devDependencies": {
     "@types/node": "^8.0.26",
+    "abstract-leveldown": "^2.7.0",
     "after": "^0.8.2",
     "async": "^2.5.0",
     "bl": "^1.2.1",
     "browserify": "^14.3.0",
     "bustermove": "~1.0.0",
     "delayed": "~1.0.1",
-    "encoding-down": "^2.1.5",
+    "encoding-down": "^2.2.0",
     "faucet": "~0.0.1",
     "leveldown": "^1.7.2",
```

Comment:
@MeirionHughes This needs to be bumped to `^1.8.0`

Reply:
fixed